packages/teams-js/src/public/video.ts

```diff
@@ -154,7 +170,19 @@ export namespace video {
     if (!isSupported()) {
       throw errorNotSupportedOnPlatform;
     }
-    registerHandler('video.effectParameterChange', callback, false);
+
+    const callbackHandler = (effectId: string | undefined): void => {
```

Comment:
`callback` or `handler`, they are basically same. but stacking both of them seems weird.

Reply:
updated